android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt

```diff
@@ -236,10 +236,26 @@ class ScreenStack(context: Context?) : ScreenContainer<ScreenStackFragment>(cont
             mTopScreen = newTop
             mStack.clear()
             mStack.addAll(mScreenFragments)
+
+            turnOffA11yUnderTransparentScreen()
+
             it.commitNowAllowingStateLoss()
         }
     }
 
+    private fun turnOffA11yUnderTransparentScreen() {
+        if (mScreenFragments.size > 1) {
+            mTopScreen?.let {
+                if (isTransparent(it)) {
+                    val belowTop = mScreenFragments[mScreenFragments.lastIndex - 1].screen
+                    belowTop.changeAccessibilityMode(IMPORTANT_FOR_ACCESSIBILITY_NO_HIDE_DESCENDANTS)
+                }
+            }
+        }
+
+        topScreen?.changeAccessibilityMode(IMPORTANT_FOR_ACCESSIBILITY_AUTO)
```

Comment:
Is this option set on all of the children when setting it on the `Screen`? If so, what if the user had different accessibility mode set on one of the children? Wouldn't we override it here?

Reply:
It is correctly handled by the `auto` setting. I've added a test for that in 078c69c to check this.